pyface/ui/qt4/code_editor/tests/test_code_widget.py

```diff
@@ -31,6 +31,11 @@ def setUpClass(cls):
     def tearDown(self):
         self.qapp.processEvents()
 
+    def test_different_lexer(self):
+        cw = CodeWidget(None, lexer="yaml")
```

Comment:
maybe add a comment about this being a test for the traitsui bug. For now, a test which just sets the correct lexer is better than no test.

Reply:
For future reference, this test failed before the fix was made, and passed after the fix.